testsuite/array_properties.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import print_function
+import unittest as ut
+import numpy as np
+import espressomd
+from espressomd import lb
+
+@ut.skipIf(not espressomd.has_features(["ROTATION", "PARTICLE_ANISOTROPY", "LANGEVIN_PER_PARTICLE", "DIPOLES", "EXTERNAL_FORCES", "LB", "EXCLUSIONS"]),
+           "Features not available, skipping test!")
```

Comment:
Please only skip the individual operations and not the entire test if a required features is not available.